build-info/github.com/raphw/byte-buddy/build.yaml

```diff
@@ -0,0 +1,4 @@
+---
+additionalArgs:
+  - "-Pextras"
+javaVersion: "8"
```

Comment:
Shouldn't this get discovered automatically? As much as possible I want to avoid having javaVersion in this repo as always going to become outdated eventually. Even some of the ones we already have in here can probably be removed now we are better at picking the java version.

Reply:
Er. Yes. Honestly not sure how that crept in and I seemed to need it. It works fine and detects JDK18 without it.